### DISCUSSION TITLE
Error with the command 'convert'

### DISCUSSION THREAD

**Discussion by Sivan3004** [ImageMagick/ImageMagick#2991] [General]
Hi,
I have an issue when I try to use the 'convert' command on linux terminal. Sometimes it works and sometimes it's not and I can't figure out the problem when it doesn't work.
Example: 
convert pre.ras pre.ras.jpg

The error:
convert-im6.q16: ImproperImageHeader `pre.ras' @error/sun.c/ReadSUNImage/496.
convert-im6.q16: no images defined `pre.ras.jpg' @error/convert.c/ConvertImageCommand/3258.

Do you have any idea what is wrong?

**Comment by snibgo** [comment]
> ImproperImageHeader pre.ras

IM is telling you that it thinks the file pre.ras has a bad header.

If you provide a link to it, perhaps someone who knows about RAS files can give you more detail.
